test-app/.eslintrc.js

```diff
@@ -24,6 +24,21 @@ module.exports = {
   },
   rules: {},
   overrides: [
+    {
+      files: ['**/*.{js}'],
+      plugins: ['ember'],
+      extends: ['eslint:recommended', 'plugin:ember/recommended'],
+    },
+    {
+      files: ['**/*.gjs'],
+      parser: 'ember-eslint-parser',
+      plugins: ['ember'],
+      extends: [
+        'eslint:recommended',
+        'plugin:ember/recommended',
```

Comment:
Do you need to specify these extends in the override if they are specified in the base config?

Reply:
Done